src/assets/scripts/client/AppController.js

```diff
@@ -147,7 +148,7 @@ export default class AppController {
 
         this.spawnPatternCollection = new SpawnPatternCollection(initialAirportData, this.navigationLibrary);
         this.spawnScheduler = new SpawnScheduler(this.spawnPatternCollection, this.aircraftController);
-        this.canvasController = new CanvasController(this.$element, this.navigationLibrary);
+        this.canvasController = new CanvasController(this.$element, this.navigationLibrary, GameController.getGameOption(GAME_OPTION_NAMES.THEME));
```

Comment:
Since `GameController` is a singleton, there is no need to pass it here.  It can simply be referenced on assignment within the `CanvasController`

Reply:
done